gapic/src/main/com/google/gapid/server/GapitPkgInfoProcess.java

```diff
@@ -23,7 +23,8 @@
 import com.google.gapid.proto.pkginfo.PkgInfo;
 import com.google.gapid.proto.pkginfo.PkgInfo.PackageList;
 
-import java.io.File;
+import java.io.*;
```

Comment:
Another .* import.

I think you can tell intellij not to do this by setting the minimum number of classes needed before it collapses imports to some very high number.